rosbag2_transport/include/rosbag2_transport/play_options.hpp

```diff
@@ -30,6 +31,12 @@ struct PlayOptions
   size_t read_ahead_queue_size;
   std::string node_prefix = "";
   float rate = 1.0;
+
+  // Topic names to whitelist when playing a bag. Only messages matching these
+  // specified topics will be played. If list is empty, the filter is ignored
+  // and all messages are played.
+  std::vector<std::string> topics_to_filter = std::vector<std::string>();
```

Comment:
```suggestion
  std::vector<std::string> topics_to_filter ={};
```

Reply:
`std::vector<std::string> topics_to_filter{}`

Reply:
Or just `std::vector<std::string> topics_to_filter{};

Reply:
just adapting to the style below for the `std::unordered_map` ;-)

Reply:
Heh, so I already changed to `{}` without the `=` so now they're inconsistent. Do we prefer changing the line below too, or change back to `={}`?